packages/ffe-file-upload-react/src/index.d.ts

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+export interface FileItem {
+    name: string;
+    document?: File;
+    error?: string;
+}
+
+export interface FileUploadProps {
+    id: string;
+    label: string;
+    files: { [key: string]: FileItem };
+    onFilesSelected(fileList: FileList): void;
```

Comment:
Antar det burde vare slik eller er dette noen syntax jag ikke er kjent med. 
```
 onFilesSelected: (fileList: FileList) => void;
 onFileDeleted( : (fileList: FileList) => void;
 onFileDeleted(: (fileList: File) => void;
 ```

Reply:
De er to måter å skrive det samme.

Reply:
OK det visste jag ikke :)